Resources/Prototypes/Datasets/Names/borg.yml

```diff
@@ -39,3 +39,9 @@
   - Cute-Bot
   - Makeshifter
   - Twin
+  - S.A.U.L
+  - Honker
+  - Clanker 
+  - Is-The-Robot
+  - 9 of 7
```

Comment:
what?

Reply:
Can you please be more specific? 
-Honker (the clown funny duh)
-Clanker (reference to battle droids from StarWars)
-Is-The-Robot (lizard name but borg)
-9 of 7 (reference to the "borg" from Star Trek they have names that are there unit number and there "batch" IE 9 of 7"

Reply:
the star trek character is 7 of 9 not 9 of 7

Reply:
Ya, its a reference so I just flipped it.